api/api.go

```diff
@@ -12,12 +12,11 @@ type API struct {
 }
 
 //Setup function sets up the api and returns an api
-func Setup(ctx context.Context, r *mux.Router) *API {
+func Setup(ctx context.Context, r *mux.Router, zc ZebedeeClient) *API {
 	api := &API{
 		Router: r,
 	}
 
-	// TODO: remove hello world example handler route
-	r.HandleFunc("/hello", HelloHandler(ctx)).Methods("GET")
+	r.HandleFunc("/releasecalendar/legacy", LegacyHandler(ctx, zc)).Methods("GET")
```

Comment:
Two comments with regards to the path:
1. By our conventions the first segment should be the plural form of the resource name. The easiest way to test whether our resource name is correct is if we can form a sentence like "The endpoint `GET /v1/{resource_name}` returns a list of `{resource_name}`" and this makes sense. So in this case the "The endpoint `GET /v1/releasecalendar` returns a list of `releasecalendar`" is incorrect. The path should therefore probably be `GET /v1/releases` as the sentence would be "The endpoint `GET /v1/releases` returns a list of `releases`.
2. It is best to include the `/v1` in your paths within the micro service. Many of our apps do not do this, but it is necessary for when we want to move to `/v2` as we can just add the second route to the API rather than having to deploy two versions of the app or some other thing. The API router has support for this pattern so I would suggest you adopt it.